packages/libs/src/sdk/api/challenges/ChallengesApi.ts

```diff
@@ -100,8 +101,8 @@ export class ChallengesApi extends BaseAPI {
       throw new Error(`Failed to find user ${args.userId}`)
     }
     const { ercWallet: recipientEthAddress, handle } = data
-    const attestationTransactionSignatures: string[] = []
-
+    const antiAbuseOracleEthAddress =
```

Comment:
would it further speed things up to not have to await for this before firing off `submitAntiAbuseOracleAttestation`?

Reply:
i was thinking this too but `getWalletAddress` is pretty instant, i think by this time SDK has already selected AAO and there's no network requests